contrib/python-bindings/source/CMakeLists.txt

```diff
@@ -21,6 +21,16 @@ INCLUDE(FindPythonLibs)
 INCLUDE(FindPythonInterp)
 
 #
+# Unset Boost_Found and run the low level FindBOOST.cmake module again to
+# pick up libboost_python.so
+#
+LIST(REMOVE_ITEM CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules/)
+SET(Boost_FOUND)
+# Use the low level _FIND_PACKAGE function instead of our wrapper
+_FIND_PACKAGE(Boost 1.54 COMPONENTS python)
```

Comment:
We cannot set a component to off. But that's a good point.
Further, this only works with external boost.